web/app/scripts/navbar/navbar-controller.js

```diff
@@ -2,45 +2,24 @@
     'use strict';
 
     /* ngInject */
-    function NavbarController($log, $rootScope, $state, $stateParams, RecordTypes) {
+    function NavbarController($log, $rootScope, $state, $stateParams,
```

Comment:
I definitely agree that some abstraction here would be beneficial. I had a couple different ideas in mind that would depend on how we'd like to interface with these values in other parts of the app. Since it's not trivial, I'm going to make an issue for it, so we can discuss the best approach and then implement it in the next Sprint.